src/components/FwbModal/FwbModal.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div>
+    <div class="bg-gray-900 bg-opacity-50 dark:bg-opacity-80 fixed inset-0 z-40" />
+    <div
+      ref="modalRef"
+      class="overflow-y-auto overflow-x-hidden fixed top-0 right-0 left-0 z-50 w-full md:inset-0 h-modal md:h-full justify-center items-center flex"
+      tabindex="0"
+      @click.self="clickOutside"
+      @keyup.esc="closeWithEsc"
+    >
+      <div
+        :class="`${modalSizeClasses[size]}`"
+        class="relative p-4 w-full h-full md:h-auto"
```

Comment:
Hi @cogor, first, I wanna thank you since I think that this library wouldn't exist or wouldn't have as much features without your work. 

Just a little question, do you think you could integrate [the change from my PR](https://github.com/themesberg/flowbite-vue/pull/226) in yours? (its a micro change, a bugfix for the modal component).

Reply:
Sure